omod-1.9/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_9/SessionController1_9.java

```diff
@@ -44,6 +52,8 @@
 @RequestMapping(value = "/rest/" + RestConstants.VERSION_1 + "/session")
 public class SessionController1_9 extends BaseRestController {
 
+	private final Log log = LogFactory.getLog(this.getClass());
```

Comment:
Our convention for the above is `private static final`
And sl4j: https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-HowToLog